client/src/rpc_client.rs

```diff
@@ -306,6 +306,35 @@ impl RpcClient {
         self.get_account(pubkey).map(|account| account.data)
     }
 
+    pub fn get_minimum_balance_for_rent_exemption(&self, data_len: usize) -> io::Result<u64> {
```

Comment:
instead of a new RPC, how about using get_account(sys::rent::id) and running the calculation locally?

Reply:
hmm. Less clutter, I like it.